mxnet/inference/docker/1.9/py3/cu112/Dockerfile.gpu

```diff
@@ -0,0 +1,116 @@
+FROM nvidia/cuda:11.2.1-cudnn8-runtime-ubuntu20.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+# Specify accept-bind-to-port LABEL for inference pipelines to use SAGEMAKER_BIND_TO_PORT
+# https://docs.aws.amazon.com/sagemaker/latest/dg/inference-pipeline-real-time.html
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+ARG MMS_VERSION=1.1.2
```

Comment:
Please change this to latest -- https://github.com/awslabs/multi-model-server/releases/tag/v1.1.4